addons/im_livechat/models/discuss_channel.py

```diff
@@ -78,6 +78,12 @@ class DiscussChannel(models.Model):
         compute="_compute_livechat_agent_providing_help_history",
         store=True,
     )
+    livechat_note = fields.Html(
+        "Livechat Note",
+        default="",
+        sanitize_style=True,
+        help="Note about the livechat channel, visible to the operator.",
+    )
```

Comment:
ACL allow reading the channel for anyone having access to the channel, so the field should have group internal on it.